### ISSUE TITLE
Invalid Attribute value at 216:50

### ISSUE THREAD

**Issue by merco** [RazrFalcon/svgcleaner#248]
What should I check in my SVG File?

[15145-A0002ToSVG-result.zip](https://github.com/RazrFalcon/svgcleaner/files/7339904/15145-A0002ToSVG-result.zip)

maybe fill-rule="winding" ?

And this token is not supported by the cleaner ?

![immagine](https://user-images.githubusercontent.com/1032335/137174775-5a0ed6ea-dabf-4f98-aee2-14ec22dd5ad0.png)

Thank you.

**Comment by RazrFalcon** [comment]
Yes, `fill-rule="winding"` is not a valid SVG. And no `@font-face` support either.
